Adding file ADOPTERS.md (#285)
kubernetes-sigs/e2e-framework: ADOPTERS.md

# E2E-Framework Adopters

> Project e2e-framework is used and loved by many in the Kubernetes community. 

Reviewer: ```suggestion
This project is used and loved by many in the Kubernetes community. 
```

Reviewer: Please check if this makes it much easier to read ?

> While we are not aware of everyon using the project, below is a list of adopters (and/or contributors) who have

Reviewer: ```suggestion
While we are not aware of everyone using the project, below is a list of adopters (and/or contributors) who have
```

agreed to share how they use the framework in their porjects.

### Adding your story

> If you are using the e2e-framework to test your Kubernetes components, and you want to share your story, feel free to open a PR and update this document.

Reviewer: ```suggestion
If you are using the `e2e-framework` to test your Kubernetes components, and you want to share your story, feel free to open a PR and update this document.
```

## E2E-Framework adopters

### Crossplane
[Crossplane](https://github.com/crossplane/crossplane) has ported all of their integration tests to use e2e-framework for their Crossplane project.
See their work [here](https://github.com/crossplane/crossplane/pull/4187)

> 

Reviewer 2: ```suggestion
[Datadog](https://github.com/DataDog/) has ported their e2e tests from the [Kubernetes e2e test framework](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/framework) to the e2e-framework. The e2e-framework is also used to test different controllers.
```
